src/visualizer/objectFunctions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @description This file contains functions that use objects, and returns objectrelated stuff
+ */
+
+(function(){
+    window.objectFunctions = {getLSValuesOfGroupNameAsKey, getstudentByStudentName, getStudentIdxInGroupByStudentName, getLSValuesOfGroup};
+    /**
+     * @summary Creates and returns a new array with the learningstyle values of the learningstyle with the learningstyle name provided in the learnStyleName argument
+     * @param {object} group the group from which to get the data
+     * @param {string} learnStyleName the name of the learningstyle e.g. "activeReflective"
+     * @returns {object} returns an object with the students of the groups values in the given learningstyle, in the format {studentname: learningstylevalue} e.g. {jens: -3}
+     */
+    function getLSValuesOfGroupNameAsKey(group, learnStyleName){
```

Comment:
Har i ikke allerede sådan en funktion?

Reply:
Den ligger i hvert fald meget tæt op ad.
Benyt den studerendes index i stedet for og så brug den anden funktion

Reply:
Done